### PR TITLE
Enochean Enhance Learn Method-2

### PR DESCRIPTION
**Pull request by hasenradball**
1. enhance learn method:
- change learn methods into one learn method for the enocean devices.
- construction with if...elif

2. Docu changes in readme and improve readibility
- docu changes in readme
- changes tab setting for device description

### CODE REVIEW

File: enocean/__init__.py [hunk 1]
```diff
@@ -764,87 +764,60 @@ def send_switch_D2(self, id_offset=0, rx_id=0, pulsew=0, on=0):
...
+        # device range 40 - 49 --> Learn protocol for other actuators
+        elif (device == 40):
+            # Eltako shutter actor FSB14, FSB61, FSB71
+             payload = [0xFF, 0xF8, 0x0D, 0x80]
+             self.logger.info('enocean: sending learn telegram for actuator with [ID-Offset], [RORG], [payload] / [{:#04x}], [{:#04x}], [{}]'.format(id_offset, rorg, ', '.join('{:#04x}'.format(x) for x in payload)))
+             return None
         else:
-            self.logger.info('enocean: sending learn telegram for radiator valve with [ID-Offset], [RORG], [payload] / [{:#04x}], [{:#04x}], [{}]'.format(id_offset, rorg, ', '.join('{:#04x}'.format(x) for x in payload)))
-            self._send_radio_packet(id_offset, rorg, payload)
+            self.logger.error('enocean: no such device defined for learning!')
```
Comment (ohinckel) [inline enocean/__init__.py]: Maybe this error message could include the missing device so that users see this in the log and can report it somewhere.

**Comment by ohinckel** [comment]
Great @hasenradball. No line ending issues anymore. So far the PR looks OK to me, but can't test all this stuff. Anyway, thanks for your contribution.

**Comment by hasenradball** [comment]
Hello it is tested bei me it works.
Device number actually added.
I have tested it actually.

`2017-12-11  09:17:20 ERROR    Main         enocean: invalid device. Device 50 actually not defined!`

Can you merge it?

**Comment by hasenradball** [comment]
Some logs from the log file:

`2017-12-11  10:38:19 INFO     Main         enocean: sending learn telegram for switch command with [ID-Offset], [RORG], [payload] / [0x00], [0xa5], [0xe0, 0x40, 0x0d, 0x80]

2017-12-11  10:38:36 INFO     Main         enocean: sending learn telegram for dim command with [ID-Offset], [RORG], [payload] / [0x00], [0xa5], [0x02, 0x00, 0x00, 0x00]

2017-12-11  10:38:43 INFO     Main         enocean: sending learn telegram for radiator valve with [ID-Offset], [RORG], [payload] / [0x00], [0xa5], [0x00, 0x00, 0x00, 0x00]

2017-12-11  10:38:52 INFO     Main         enocean: sending learn telegram for actuator with [ID-Offset], [RORG], [payload] / [0x00], [0xa5], [0xff, 0xf8, 0x0d, 0x80]

2017-12-11  10:39:21 ERROR    Main         enocean: sending learn telegram with invalid device! Device 5 actually not defined!
`

**Comment by hasenradball** [comment]
see log examples now:
`2017-12-11  11:15:15 INFO     Main         enocean: sending learn telegram for radiator valve with [Device], [ID-Offset], [RORG], [payload] / [30], [0x00], [0xa5], [0x00, 0x00, 0x00, 0x00]`

`2017-12-11  11:15:19 INFO     Main         enocean: sending learn telegram for actuator with [Device], [ID-Offset], [RORG], [payload] / [40], [0x00], [0xa5], [0xff, 0xf8, 0x0d, 0x80]`

**Comment by ohinckel** [comment]
Thanks for the update!

**Comment by hasenradball** [comment]
Hello ohinckel,
Thank you for your support.
The next step I want to do is to merge the enochean file of the develop with the master. To get the benefit of the actuator which aschwith has merged to the master.
Is there a smart way doing it?

**Comment by msinn** [comment]
@hasenradball Do I understand you right: There is functionallity in the master that isn‘t in the develop??? That shouldn‘t happen at all!

**Comment by hasenradball** [comment]
Hello msinn, I am sorry it was a miss information of me. I checked it already now.
The change was also in the develop before.
It was the last update in the enocean (in master).
I found it also in the develop.
Sorry.

**Pull request merged**